labgrid/driver/powerdriver.py

```diff
@@ -182,26 +182,35 @@ def get(self):
 class YKUSHPowerDriver(Driver, PowerResetMixin, PowerProtocol):
     """YKUSHPowerDriver - Driver using a YEPKIT YKUSH switchable USB hub
         to control a target's power - https://www.yepkit.com/products/ykush"""
-    bindings = {"port": "YKUSHPowerPort", }
+    bindings = {"port": {"YKUSHPowerPort", "NetworkYKUSHPowerPort"}, }
     delay = attr.ib(default=2.0, validator=attr.validators.instance_of(float))
 
-
     def __attrs_post_init__(self):
         super().__attrs_post_init__()
-        # uses the YKUSH pykush interface from here:
-        # https://github.com/Yepkit/pykush
-        self.pykush_mod = import_module('pykush.pykush')
-        self.pykush = self.pykush_mod.YKUSH(serial=self.port.serial)
+        if self.target.env:
+            self.tool = self.target.env.config.get_tool('ykush') or 'ykushcmd'
+        else:
+            self.tool = 'ykushcmd'
+
+    def _get_ykushcmd_prefix(self):
+        prefix = self.port.command_prefix+[
+            self.tool,
+            "-s", str(self.port.serial),
```

Comment:
Does the ykushcmd tool talk to all connected devices to find the one with the correct serial or does it use the USB descriptor information? Otherwise, one could get problems with concurrent accesses (which probably also exist already with pykush).